vterm.el

```diff
@@ -410,6 +410,12 @@ The units are seconds.")
            :sentinel (when (or vterm-exit-functions
                                vterm-kill-buffer-on-exit)
                        #'vterm--sentinel))))
+
+  ;; Change major-mode is not allowed
+  (add-hook 'change-major-mode-hook
+            #'(lambda () (interactive)
```

Comment:
Don't think it matters much but you don't have to sharp quote lambda.

Reply:
Fixed. Thanks.

Reply:
Sorry. I meant lambda's are self quoting and you shouldn't quote them at all.
A good read is https://nullprogram.com/blog/2017/12/14/